spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Recurrent.scala

```diff
@@ -258,7 +256,7 @@ class Recurrent[T : ClassTag](var batchNormParams: BatchNormParams[T] = null)
     cells(times - 1).output.toTable(hidDim)
   }
 
-  private var initState: Activity = null
+  protected var initState: Activity = null
   def setState(state: Activity): Unit = {
```

Comment:
who will call setState?

Reply:
In our current scenario, it will be used in decoder if there is only one cell. Besides getState/setState is a pair. we provide setState in case some user want to use a pretrained init state instead of zeros in encoder.

Reply:
Who will call that even for decoder?

Reply:
My plan is giving a seq2seq layer, it will call setState for decoder.